### DISCUSSION TITLE
Not appearing as Wordpress Element (HTML Block)

### DISCUSSION THREAD

**Discussion by alfandango** [google/model-viewer#4175] [Q&A]
I have taken the code snippet from the editor and uploaded this to a static site on my domain - It works fine.

I have then used that code snippet in a wordpress block on the same domain (all links adjusted to point to above location) but nothing appears on the Wordpress page, the buttons appear for a split second, then disapear but no model.

Any suggestions for troubleshooting this?

**Comment by elalish** [comment]
Check the JS console. Best guess is a CORS problem: https://modelviewer.dev/docs/faq.html#entrydocs-technical-questions-cors
